test/XRP/fakes/fake-common-xrpl-client.ts

```diff
@@ -3,19 +3,25 @@ import { Wallet } from '../../../src/index'
 import Result from '../../Common/Helpers/result'
 import { XrplNetwork } from 'xpring-common-js'
 import RawTransactionStatus from '../../../src/XRP/raw-transaction-status'
-import TransactionResult from '../../../src/XRP/model/transaction-result'
+import TransactionResult from '../../../src/XRP/model/final-transaction-result'
 
 class FakeCommonXrplClient implements CommonXrplClientInterface {
   public constructor(
-    public awaitFinalTransactionStatusValue: Result<RawTransactionStatus>,
+    public awaitFinalTransactionStatusValue: Result<{
+      rawTransactionStatus: RawTransactionStatus
+      lastLedgerPassed: boolean
```

Comment:
might be useful to make a named type for this imo. alternatively, i guess you could do `ReturnType< typeof CommonXrpClientInterface.awaitFinalTransactionStatusValue>`

Reply:
mm yeah I thought about that too... maybe I can make a little class right there in the `Core` client